### DISCUSSION TITLE
Setting Canvas Dimensions to "fit"

### DISCUSSION THREAD

**Discussion by Bill-VA** [SVG-Edit/svgedit#795] [New feature request]
I want the exported SVG to auto resize depending on the available window size, but when I choose "fit to content" from the select box in the Canvas Dimensions option of Image Properties, after taking 30-60 seconds to process, the dialog box disappears and the chosen "fit to content" setting did not "stick".

BTW, I like how your editor assigns ID values to the svg elements.

**Comment by Bill-VA** [comment]
Thought I'd leave a follow up comment.  If I edit the first line of the SVG, to read like this:

`<svg width="100%" height="100%" viewBox="0 0 1023 790" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg" xmlns:se="http://svg-edit.googlecode.com" data-name="Layer 1">`

where 1023 x 790 is the size of the image, the svg will now resize accordingly.

**Comment by inspector71** [comment]
I think this menu option is only meant to apply to editing mode. That is,
how you view the item whilst it's being edited. Not how the item behaves
when eventually used "in the wild".

**Comment by inspector71** [comment]
Of topic ... That xmlns:se URL should really be updated to GitHub.

**Comment by brettz9** [comment]
Thanks for the input, @inspector71 . I think I'll change this to enhancement then accordingly...

Re: the xmlns:se namespace... Namespaces don't actually need to resolve to an up-to-date URL (and XML tools *shouldn't* normally try to resolve them), though obviously that can be convenient for finding docs and such... I'd worry about breaking old code...

**Comment by brettz9** [comment]
FWIW, you may also be interested in issue #267 if you hadn't seen that...

**Comment by inspector71** [comment]
Fair enough Brett. Makes sense to be cautious.
